node/src/EnhancedEventEmitter.ts

```diff
@@ -47,4 +52,39 @@ export class EnhancedEventEmitter extends EventEmitter
 			}
 		});
 	}
+
+	on<K extends keyof E & string>(event: K, listener: (...args: E[K]) => void)
+	{
+		super.on(event, listener as (...args: any[]) => void);
+
+		return this;
+	}
+
+	off<K extends keyof E & string>(event: K, listener: (...args: E[K]) => void)
```

Comment:
IMHO there is no single place where we use `off()`

Reply:
But users of the library might want to unsubscribe from events

Reply:
That's right ⬆️ , and it does not do any harm, it's a mere alias. :+1:

Reply:
For completeness I added `prepend(Once)Listener` as well.

Btw. there is `safeEmitAsPromise` which seems to be not used internally, but removing it would be a breaking change.